File: Android/blackbox/src/main/java/com/hhd2002/blackbox/AzureTableSessionEntity.java
package com.hhd2002.blackbox;

import com.microsoft.azure.storage.table.TableServiceEntity;

public class AzureTableSessionEntity extends TableServiceEntity {
    public AzureTableSessionEntity() {
        this.partitionKey = "PartitionKey";
    }

    private String SessionStr;

    public String getSessionStr() {
        return this.SessionStr;
    }

    public void setSessionStr(String sessionStr) {
        this.SessionStr = sessionStr;
    }
}

File: DotNet/BlackBoxLib/AzureTableSessionEntity.cs
using Microsoft.WindowsAzure.Storage.Table;

namespace BlackBoxLib
{
    public class AzureTableSessionEntity : TableEntity
    {
        public AzureTableSessionEntity()
        {
            this.PartitionKey = "PartitionKey";
        }

        public string SessionStr { get; set; }
    }





























}
